test/project_types/script/layers/domain/extension_point_test.rb

```diff
@@ -34,6 +34,10 @@
   end
 
   describe ".new" do
+    before do
+      ShopifyCLI::Feature.stubs(:enabled?).with(:scripts_beta_languages).returns(false)
+    end
```

Comment:
Yes. Certain tests were failing depending on the status of my environment, because the wasm language was being added outside of the normal config process. 
The tests env should not be caring about my dev env.

That being said, since we're back on sparse checkout, I moved the wasm config to be part of the normal .yml file, so we don't need this anymore.